optaplanner-persistence/optaplanner-persistence-jaxb/src/test/java/org/optaplanner/persistence/jaxb/api/score/AbstractScoreJaxbXmlAdapterTest.java

```diff
@@ -71,7 +71,8 @@
                     + "<([\\w\\-\\.]+)/>"; // Start and end of element
         }
         if (!xmlString.matches(regex)) {
-            fail("Regular expression match failed.\nExpected regular expression: " + regex + "\nActual string: " + xmlString);
+            fail(String.format("Regular expression match failed.%nExpected regular expression: " + regex +
+                                       "%nActual string: " + xmlString));
```

Comment:
If you use `format`, don't use concatenate `+`: `format("... Expected regular expression (%s) ...", regex)`